src/wallet/wallet.ts

```diff
@@ -539,7 +539,7 @@ class HathorWalletServiceWallet extends EventEmitter implements IHathorWallet {
     let shouldGetNewAddresses = false;
 
     for (const output of outputs) {
-      if (this.newAddresses.find(newAddress => newAddress.address === output.decoded.address)) {
```

Comment:
This was a bug, we were not generating new addresses when a new tx came from the websocket.